### DISCUSSION TITLE
Is it possible to create a custom frontpage ?

### DISCUSSION THREAD

**Discussion by erdembey** [CaiJimmy/hugo-theme-stack#1039] [Q&A]
Hello,

Title is clear but my intention is to create custom frontpage to able to provide basic static information and maybe featured / latest items from my different post types.

Is it possible to create such a frontpage ?

Thanks.

**Comment by CaiJimmy** [comment]
Yes, you can create a `content/_index.md` file.

**Comment by AmanRathoreP** [comment]
I also added a custom front page and changed the default home page of hugo to Explore of my site.
you can view that [here](https://amanr.me/)
